src/app_service/service/wallet_account/service_token.nim

```diff
@@ -166,6 +169,7 @@ proc checkRecentHistory*(self: Service, addresses: seq[string]) =
     error "error: ", errDescription
 
 proc reloadAccountTokens*(self: Service) =
+  self.events.emit(SIGNAL_WALLET_RESET_RELOAD_TIMER, Args())
   let addresses = self.getWalletAddresses()
   self.buildAllTokens(addresses, store = true)
   self.checkRecentHistory(addresses)
```

Comment:
Forgot to say, from here, just call `RestartWalletReloadTimer` from the statusgo side, which you need to add there, left a comment about it in your PR there.